code/modules/admin/create_poll.dm

```diff
@@ -0,0 +1,126 @@
+/client/proc/create_poll()
+	set category = "Special Verbs"
+	set name = "Create Poll"
+
+	if(!check_rights(R_ADMIN|R_DEBUG|R_DEV))
```

Comment:
Might end up regretting `R_DEV` here.

Though, if this proc is only given to admins, why are the other two flags even checked?